src/uglify/worker.js

```diff
@@ -1,9 +1,14 @@
 import minify from './minify';
-import { decode } from './serialization';
 
-module.exports = (options, callback) => {
+/* eslint-disable no-new-func */
+
+module.exports = (workerOptions, callback) => {
```

Comment:
`workerOptions` => `options` (Just reassign `options` them below please)

Reply:
@michael-ciniawsky eslint throw problem about `no-param-reassign`, but it is not make sense here.

Reply:
`eslint-disable-next-line` ? 🙃